gramps/gen/plug/utils.py

```diff
@@ -361,9 +356,9 @@ def load_addon_file(path, callback=None):
         return False
     fptr.close()
     # file_obj is either Zipfile or TarFile
-    if path.endswith(".zip") or path.endswith(".ZIP"):
+    if path.endswith((".zip", ".ZIP")):
```

Comment:
```suggestion
    if path.lower().endswith((".zip")):
```
would this be ever clearer and handle mixed case (.Zip etc.) correctly as well?